include/dsn/internal/ports.h

```diff
@@ -123,7 +123,7 @@ namespace dsn
 
 # ifdef _WIN32
 
-// TODO(qinzuoyan) port to windows
+// TODO(qinzuoyan) port to big-endian machine
```

Comment:
I didn't follow you. Big-endian byte order is also referred to as network
byte order, so I think be32toh is functionally equivalent to ntoh.

On Tue, Jun 21, 2016 at 10:23 AM, Zhenyu Guo <notifications@github.com>
wrote:

> In include/dsn/internal/ports.h
> <https://github.com/imzhenyu/rDSN/pull/403#discussion_r67799093>:
>
> > @@ -123,7 +123,7 @@ namespace dsn
> >
> >  # ifdef _WIN32
> >
> > -// TODO(qinzuoyan) port to windows
> > +// TODO(qinzuoyan) port to big-endian machine
>
> can we use ntoh and hton for this purpose?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/imzhenyu/rDSN/pull/403/files/f6ca9bff58d152a61bad7f9c6641bf8ab60bf181..176a36f5435a494dd31241608277be58ec66813d#r67799093>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAgQPcU-I8RFiX4CCiyvhJEzhMXbGssKks5qN0s7gaJpZM4I2J8D>
> .
>